apps/examples/client/example_view.coffee

```diff
@@ -0,0 +1,26 @@
+Backbone = require 'backbone'
+ExampleChannelView = require './example_channel_view.coffee'
+ 
+template = -> require('../templates/example.jade') arguments...
+
+module.exports = class ExampleView extends Backbone.View
+
+  events: 
+    'click .js-load-more' : 'fetchAndRender'
+
+  initialize: ->
+    @initializeSubViews()
+
+  initializeSubViews: ->
+    @$('.ChannelBlockGroup').each ->
+      new ExampleChannelView
+        el: $(this)
+
+  fetchAndRender: (e) ->
+    e.preventDefault()
+    
+    id = @$el.data('id')
+    
+    $.get "/api/examples/#{id}", (example) =>
+      @$el.html template
+        example: example
```

Comment:
This whole thing doesn't look like a view to me either actually. Would be much simpler to read as just normal JavaScript. You're sort of just using views as a way to scope selectors which makes this extremely confusing. Just scope it and that's that.